samples/sample-sync-doc-definitions.js

```diff
@@ -10,7 +10,6 @@ function() {
 
   // Creates a RegExp to match the ID of an entity that belongs to a business
   function createBusinessEntityRegex(suffixPattern) {
-    // While business IDs have traditionally been numeric only, this
```

Comment:
After running a diff on the tool's output both before and after issue #66's implementation, I can confirm that this is the only non-whitespace change to the generated sync function.